### ISSUE TITLE
Dependency pysnmp is incompatible with Python 3.12

### ISSUE THREAD

**Issue by jelly** [threat9/routersploit#819]
## Steps to Reproduce (for bugs)
1. Run tests under Python 3.12

## Your Environment
* RouterSploit Version used: 3.4.2
* Operating System and version: Arch 
* Python Version: ( `python3 --version`)  3.12
* Python Environment: ( `python3 -m pip freeze` )

## Current Behavior

routersploit depends on pysnmp, but pysnmp still uses `asyncore` which has been removed in Python 3.12 and there seems to be no activity in the pysnmp repository for quite some time.

## Expected Behavior

Switch to a different library or drop the functionality? Up to the maintainer.

**Comment by cybaol** [comment]
@jelly 
[New pysnmp upstream](https://github.com/lextudio/pysnmp) which forked from [etingof/pysnmp](https://github.com/etingof/pysnmp).

**Comment by jelly** [comment]
Yes, I think we will switch our packaging over to this version, thanks!

**Comment by lucyoa** [comment]
Moved to the new pysnmp version
